toxav/toxav.c

```diff
@@ -157,9 +158,7 @@ ToxAV *toxav_new(Tox *tox, Toxav_Err_New *error)
 
     // TODO(iphydf): Don't rely on toxcore internals.
     Messenger *m;
-    //!TOKSTYLE-
-    m = *(Messenger **)tox;
-    //!TOKSTYLE+
+    m = tox->m;
```

Comment:
Nit: I'd collapse this assignment into the line above.

Reply:
Me too, but it's not possible due to the use of `goto`.